.travis.yml

```diff
@@ -55,9 +55,10 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       npm test;
       ./scripts/ci/ensure-staged-sass.sh;
+      node ./scripts/ci/ensure-angular-dependencies-compatibility.js;
```

Comment:
As we now have two "post build consistency checks" maybe they should just both get run by another npm `scripts` target in `package.json`?
Then we wouldn't need to edit both `.travis.yml` and `appveyor.yml` in future when adding additional checks, only `package.json` :-)